### PR TITLE
Ch2 delete .view(-1) after torch.max() in train() function

### PR DESCRIPTION
**Pull request by MarcusFra**
I think in the line 
`correct = torch.eq(torch.max(F.softmax(output, dim=1), dim=1)[1], targets).view(-1)` 
applying the `.view(-1)` function is not required since `torch.eq(torch.max(F.softmax(output, dim=1), dim=1)[1], targets)` is a 1D tensor and thus applying `view()` does not change the shape of the tensor. Am I wrong or is there another reason for the usage?

### CODE REVIEW

**Comment by falloutdurham** [comment]
I think that's right!
